vizro-core/examples/default/app.py

```diff
@@ -516,22 +526,20 @@ def create_home_page():
     return page_home
 
 
-create_country_analysis()
-
 dashboard = vm.Dashboard(
     pages=[
         create_home_page(),
         create_variable_analysis(),
         create_relation_analysis(),
         create_continent_summary(),
-        # create_country_analysis(),
+        create_country_analysis(),
     ],
-    navigation=vm.Navigation(
-        pages={
-            "Analysis": ["Homepage", "Variable Analysis", "Relationship Analysis", "Country Analysis"],
-            "Summary": ["Continent Summary"],
-        }
-    ),
+    # navigation=vm.Navigation(
+    #     pages={
+    #         "Analysis": ["Homepage", "Variable Analysis", "Relationship Analysis", "Country Analysis"],
+    #         "Summary": ["Continent Summary"],
+    #     }
+    # ),
```

Comment:
I guess this is just for testing, but don't forget to clean it up when merging :)